Wireframe/index.html

```diff
@@ -4,10 +4,63 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="../Wireframe/css/style.css">
   </head>
+ 
   <body>
+
     <!-- Add your HTML markup here -->
+    <!-- Add your HTML markup here -->
+    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+   <header>
+      
+        <h1 class="header">Learning how to use Git</h1>
+        <p class="heading_summary">This page will help you develop some understanding of Git</p>
+   </header>
+
+      <main class="container">
+
+            <article class="article" id="article-1">
+         
+            <h1 class="article-heading">What is Git?</h2>
```

Comment:
consider using article h1 in css and then remove the class article-heading as not required